From 0ac257acc24f7e75d225c9368d35dc0a1d4386fe Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Tue, 15 May 2012 14:50:10 +0300
Subject: [PATCH 178/311] improve documentation of c2_confc_ctx::fc_ast

Explain who uses c2_confc_ctx::fc_ast and what for.

LogD: #1310
RB: r/714
---
 conf/confc.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/conf/confc.h b/conf/confc.h
index 1be974d..008d0e5 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -398,7 +398,10 @@ struct c2_confc_ctx {
 	struct c2_confc         *fc_client;
 	/** Confc state machine. */
 	struct c2_sm             fc_mach;
-	/** Asynchronous system trap. */
+	/**
+	 * Asynchronous system trap, used by the implementation to
+	 * schedule a transition of ->fc_mach state machine.
+	 */
 	struct c2_sm_ast         fc_ast;
 	/** Origin of the path. */
 	struct c2_conf_obj      *fc_origin;
-- 
1.8.3.2

